### PR TITLE
docs(wizard): remember the last step on close

### LINKED ISSUE #1845: Wizard state not resetting on close
Can we add a function to reset the wizard state? When closing the wizard & reopening, the same step I am on when I initially closed the wizard is shown.

### PR DESCRIPTION
**Pull request by boaz0**
**What**:

Adding an example of how to remember the step on closing the wizard.
This fixes #1845 

//cc @ibolton336

### CODE REVIEW

File: packages/patternfly-4/react-core/src/components/Wizard/Wizard.md [hunk 1]
```diff
@@ -232,3 +233,66 @@ class SimpleWizard extends React.Component {
...
+    ];
+
+    return (
+      <React.Fragment>
+        <Button variant="primary" onClick={this.toggleOpen}>
+          Show Wizard
+        </Button>
+        {isOpen && (
+          <Wizard
+            startAtStep={this.state.step}
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/src/components/Wizard/Wizard.md]: Can you destructure this with isOpen at line 266.

**Comment by boaz0** [comment]
Hi @tlabaj 
After reading @ibolton336 comment in #1845 I updated the PR so now we have a `curStep` prop that will determine whether to reset the step or not when `wizard` is closed.
Would you mind taking a second look?
Thank you so much.

**Comment by boaz0** [comment]
OK I am going to revert those changes I have made and return to the previous one.

**Comment by boaz0** [comment]
@tlabaj & @dlabaj can you review this plz :smiley:

File: packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ export interface WizardProps {
   className?: string;
   /** The wizard steps configuration object */
   steps: WizardStep[];
-  /** The step to start the wizard at (1 or higher) */
+  /** The current step the wizard at (1 or higher) */
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx]: can you update this to say "  The step to start the wizard **is** at"

**Pull request merged**